phpunit/style-engine/style-engine-test.php

```diff
@@ -493,6 +493,25 @@ public function data_wp_style_engine_get_styles() {
 					),
 				),
 			),
+
+			'inline_background_image_url_with_background_size' => array(
+				'block_styles'    => array(
+					'background' => array(
+						'backgroundImage' => array(
+							'url' => 'http://example.com/image.jpg',
+						),
+						'backgroundSize'  => 'cover',
+					),
+				),
+				'options'         => array(),
+				'expected_output' => array(
+					'css'          => "background-image:url('http://example.com/image.jpg');background-size:cover;",
+					'declarations' => array(
+						'background-image' => "url('http://example.com/image.jpg')",
```

Comment:
I wonder if we should use `https` in the examples 😀

Reply:
Good call! Done.